dtd-catalog-exporter/src/services/mongodb-e-service.service.ts

```diff
@@ -0,0 +1,131 @@
+import { MongoClient } from "mongodb";
+import { env } from "../configs/env.js";
+import { EService, eserviceSchema, Attribute, attributeSchema, Tenant, tenantSchema } from "../models/index.js";
+
+export class MongoDBEServiceClient {
+  private client: MongoClient;
+
+  private constructor(client: MongoClient) {
+    this.client = client;
+  }
+
+  /**
+   * Connects to the mongodb database
+   */
+  public static async connect() {
+    const connectionString = `mongodb://${env.READ_MODEL_DB_USER}:${env.READ_MODEL_DB_PASSWORD}@${env.READ_MODEL_DB_HOST}:${env.READ_MODEL_DB_PORT}?directConnection=true&readPreference=secondaryPreferred&retryWrites=false`;
```

Comment:
Note that these connection parameters work just when running from local with the tunnel.
When deployed the parameters should be `?replicaSet=rs0&readPreference=secondaryPreferred`